load_test.go

```diff
@@ -338,6 +338,30 @@ points:
 	}
 }
 
+func TestTemplateFunc(t *testing.T) {
+	const configFile = "/tmp/conf-json-test.yml"
```

Comment:
Don't need to change this, but neat func to make it more portable:

https://golang.org/pkg/io/ioutil/#TempFile